todoman/ui.py

```diff
@@ -97,7 +96,15 @@ def __init__(self, todo, lists, formatter):
             + '\n'.join(' {}: {}'.format(k, v) for k, v
                         in widgets.ExtendedEdit.HELP)
         )
-
+        
+        def change_current_list(new_list):
```

Comment:
Turns out that this function needs to accept two parameters before `new_list`, called `radio_button` and `new_state`.